packages/gestalt/src/Sheet.js

```diff
@@ -1,321 +1,90 @@
-                          onClick={onDismiss}
-                        />
-                      </Box>
-                    </Flex>
-                    {subHeading}
-                  </Box>
-                )}
-                {!heading && (
-                  <Box display="flex" flex="grow" justifyContent="end" marginBottom={8}>
-                    <Box
-                      flex="none"
-                      paddingX={6}
-                      paddingY={7}
-                      position="absolute"
-                      zIndex={new FixedZIndex(1)}
-                    >
-                      <DismissButton
-                        accessibilityDismissButtonLabel={accessibilityDismissButtonLabel}
-                        onClick={onDismiss}
-                      />
-                    </Box>
-                  </Box>
-                )}
-                <ScrollBoundaryContainerProvider>
-                  <InternalScrollBoundaryContainer
-                    onScroll={updateShadows}
-                    padding={PADDING_BOINTS}
-                    ref={contentRef}
-                  >
-                    {children}
-                  </InternalScrollBoundaryContainer>
-                </ScrollBoundaryContainerProvider>
-                {Boolean(footer) && (
-                  <Box
-                    borderStyle={showBottomShadow ? 'raisedBottomShadow' : undefined}
-                    position="relative"
-                    fit
-                  >
-                    <Box padding={PADDING_BOINTS}>{footer}</Box>
-                  </Box>
-                )}
-              </Box>
-            </div>
-          </Backdrop>
-        </div>
-      </TrapFocusBehavior>
-    </StopScrollBehavior>
-  );
-}
-
-/**
- * <AnimatedSheet> component: adds animation capabilities
- */
-
 /**
  * [Sheets](https://gestalt.pinterest.systems/web/sheet ) are surfaces that allow users to view optional information or complete sub-tasks in a workflow while keeping the context of the current page. The most common example of Sheet displays content in a panel that opens from the side of the screen for the user to read or input information. Sheets have default, internal padding for content.
  *
  * ![Sheet light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Sheet.spec.mjs-snapshots/Sheet-chromium-darwin.png)
  * ![Sheet dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Sheet-dark.spec.mjs-snapshots/Sheet-dark-chromium-darwin.png)
  */
-function AnimatedSheet(props: AnimatedSheetProps): Node {
-  const {
-    accessibilityDismissButtonLabel,
-    accessibilitySheetLabel,
-    children,
-    closeOnOutsideClick,
-    onAnimationEnd,
-    onDismiss,
-    footer,
-    heading = undefined,
-    size,
-    subHeading = undefined,
-  } = props;
-
+function Sheet({
+  accessibilityDismissButtonLabel,
+  accessibilitySheetLabel,
+  children,
+  closeOnOutsideClick = true,
```

Comment:
We should really update this prop name at some point — boolean props that default to `true` is a bit of an anti-pattern. This should really be something like `disableCloseOnOutsideClick`. (Totally out of scope for this PR!)